### PR TITLE
mce: Update config file to use udev based battery tracking.

### PR DESCRIPTION
**Pull request by MagneFire**
This PR is part of https://github.com/AsteroidOS/meta-asteroid/pull/61

Big thanks to @drohhyn for finding this issue :+1:

### CODE REVIEW

File: recipes-nemomobile/mce/mce/10mce.ini [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 [Modules]
 
-# Path to modules
+# Path to modules, usually architecture dependant
```
Comment (FlorentRevest) [inline recipes-nemomobile/mce/mce/10mce.ini:5]: architecture? you mean watch ? or soc ?

File: recipes-nemomobile/mce/mce/10mce.ini [hunk 2]
```diff
@@ -16,7 +16,7 @@ ModulePath=/usr/lib/mce/modules
 # to avoid unnecessary brightness fluctuations on mce startup
 #
 # Note: the name should not include the "lib"-prefix
-Modules=radiostates;display;keypad;led;battery-statefs;inactivity;alarm;callstate;audiorouting;powersavemode;cpu-keepalive;doubletap;packagekit;sensor-gestures;bluetooth;memnotify;usbmode
+Modules=radiostates;display;keypad;led;battery-udev;inactivity;alarm;callstate;audiorouting;powersavemode;cpu-keepalive;doubletap;packagekit;sensor-gestures;bluetooth;memnotify;usbmode
```
Comment (FlorentRevest) [inline recipes-nemomobile/mce/mce/10mce.ini:19]: I guess I'm lacking context here. I thought you were replacing statefs on all watches, why is this patch only for mtk6580 ?

File: recipes-nemomobile/mce/mce/10mce.ini (hunk 1 above)
Comment (MagneFire) [reply recipes-nemomobile/mce/mce/10mce.ini:5]: Honest answer, I don't now. This is what upstream did so I just copied that part too 
https://github.com/AsteroidOS/mce/pull/9/files#diff-3c3643a28be8621cbc0d0738040f0050a32a4bb983481e7344cbb4526d23369eR5 and https://git.sailfishos.org/mer-core/mce/blob/master/inifiles/mce.ini#L5

File: recipes-nemomobile/mce/mce/10mce.ini (hunk 2 above)
Comment (MagneFire) [reply recipes-nemomobile/mce/mce/10mce.ini:19]: mce.ini is by default provided by `meta-asteroid` as in, it's copied from the repository.
Apparently, the mtk6580 hybris layer uses a custom config file. As such it needs to be altered :wink: 

Relevant upstream commit: https://git.sailfishos.org/mer-core/mce/commit/d2e2e0aa74aecce32c98d80f457dcff5234e880e

Comment (FlorentRevest) [reply recipes-nemomobile/mce/mce/10mce.ini:19]: oh I see, I hope I wrote a more descriptive commit desc in https://github.com/AsteroidOS/meta-mtk6580-hybris/commit/2ed980214aa981ee7d4ddae75d03d12efb4ce4a9
LGTM then

**Pull request merged**